src/lib/Microsoft.Health.Fhir.Ingest.Template/Microsoft.Health.Fhir.Ingest.Template.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
We use this package inside of `health-pass`. AFAIK, `health-pass` still uses .net framework 4.7.2. The reason that matters is I don't think net472 supports .net core 6.0. We might need to keep this as netstandard2.0 so that it can continue to be used in `health-pass`.

https://dotnet.microsoft.com/en-us/platform/dotnet-standard

Reply:
Connected offline with Rob about this - submitting an update soon

Reply:
Sent out new commit that leaves all netstandard2.0 projects so that they can be used with net472 or .net6